### ISSUE TITLE
more colors?

### ISSUE THREAD

**Issue by ambr0sial** [billythegoat356/pystyle#20]
okay so i have two questions actually

does an available colours list exists?

can you add more colors in the future, like purple and so?

**Comment by billythegoat356** [comment]
Purple exist!
If you look in the code of the module you can see all the already made colors. If you want more colors you can create them with StaticMIX / DynamicMIX ;)

**Comment by CColdFox** [comment]
Hello again, and thank you for your answer!

Is it possible to create a color with RGB? I think I need to use StaticRGB(), but I don't know how to use it, like for example I want to have a purplish colour (I know purple already exists, but I want to create a specific purple), I did that:
`StaticRGB(149,133,255)` but it just shows up as white. What can I do?

**Comment by billythegoat356** [comment]
Hey, that's weird. StaticRGB works very well on my PC, I don't understand why it doesn't on yours. I'll look into it.

**Comment by CColdFox** [comment]
Yeah I don't know why it does that. By the way, I tried updating PyStyle just in case, but it's already at 2.0, which must be the latest version I guess?
